### DISCUSSION TITLE
(ANDROID) Osu lazer mania

### DISCUSSION THREAD

**Discussion by maxdp66** [ppy/osu#20892] [Ideas]
**Type**
Game behaviour

**Request**
When playing Osu! Lazer (mania) on android, enable turning the phone in portrait mode for easier button mashing.

**Screenshot** (I'm bad at photoshop, I hope you get what I mean)
![Screenshot_2022-10-23-17-05-19-39_25a1a32208bbcdc1d450b7aa854bb161 (1)](https://user-images.githubusercontent.com/22943253/197400636-227a8bd4-64f3-4836-9acb-93a207e0439f.jpg)
![Screenshot_2022-10-23-17-05-19-39_25a1a32208bbcdc1d450b7aa854bb161](https://user-images.githubusercontent.com/22943253/197400689-03b053a0-b717-4e77-a0b2-117ba3b4d56d.png)

**Comment by nagi-desuuu** [comment]
There have been numerous issues exactly like the one you just put out. See https://github.com/ppy/osu/discussions/16783
**Please Search Before Posting!**

**Reply by maxdp66** [reply]
Not really an issue, more like an idea/request but I get it. Thanks.
